### ISSUE TITLE
Erroneous release v2.4.1

### ISSUE THREAD

**Issue by aasseman** [openebs/zfs-localpv#503]
- The Helm files point to container image `openebs/zfs-driver:2.4.0` instead of `openebs/zfs-driver:2.4.1`
- Container image `openebs/zfs-driver:2.4.1` isn't even published.

I was hoping to get the https://github.com/openebs/zfs-localpv/pull/451 fix, but since the latest Helm release still points to the older code it's not.

**Comment by jnels124** [comment]
@aasseman I have worked around this by publishing my own image based off the 2.4.1 tag. You can use that image if you wish:

  ```
zfsPlugin:
    image:
      registry: gcr.io/
      repository: mirrornode/zfs-driver
      pullPolicy: IfNotPresent
      tag: 2.4.1-HEDERA
```
Depending on your cluster setup, you may also need to add.

 ```
 crd:
    volumeSnapshot: false
```

**Comment by niladrih** [comment]
Hi @aasseman. The patch releases on the helm chart for installer changes. Code changes go into minor releases. There's a planned release in the next few weeks, I think we'd be able to roll out '2.5.0' soon.

**Comment by Abhinandan-Purkait** [comment]
@niladrih Now that 2.5.0 is released, can we close this issue?

**Comment by niladrih** [comment]
Fixed with v2.5.0 release.
